tx_list.go

```diff
@@ -338,3 +338,10 @@ func (tx *Tx) CheckExpire(bucket string, key []byte) bool {
 	}
 	return false
 }
+
+func (tx *Tx) GetListTTL(bucket string, key []byte) (uint32, error) {
+	if err := tx.checkTxIsClosed(); err != nil {
+		return 0, err
+	}
+	return tx.db.ListIdx[bucket].GetListTTL(string(key))
```

Comment:
Hi, Seems it shows this line is not covered by the unit test. Would you like to add a unit test to cover it? Thanks in advance.

Reply:
Already commited based on your suggestion, thanks！